whois-api/src/test/java/net/ripe/db/whois/api/mail/dequeue/MessageServiceTestIntegration.java

```diff
@@ -24,6 +28,14 @@ public class MessageServiceTestIntegration extends AbstractMailMessageIntegratio
     @Autowired
     private MessageService messageService;
 
+    final static String PASSWORD_WARN =
+            """
+            Warning: Password authentication will be removed from Mailupdates in a future
+                        Whois release as the mail message may have been sent insecurely.
+                        Please switch to PGP signing for authentication or use a different
```

Comment:
Why not use the UpdateMessages?

Reply:
Just tests and if I use that message I get an unformatted string
all the string is in one line